tests/integration/istiodremote/base/main_test.go

```diff
@@ -0,0 +1,40 @@
+// +build integ
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package base
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/tests/integration/istiodremote"
+)
+
+var ist istio.Instance
+
+func TestMain(m *testing.M) {
```

Comment:
why do we have istiodremote/base and istiod/multicluster? Not clear what these do differently without documentation

Reply:
I think with this change we can get rid of the `tests/integration/multicluster` package. It doesn't provide further coverage than the individual multi-cluster jobs; we were mostly keeping it around for the centralremotekubeconfig/ test.

Reply:
Looking at the `multicluster/base` test, I see it has a number of tests in addition to the `reachability` test Frank added in `istiodremote/multicluster`. I'm not sure if Frank tried to pull the other tests over or not. I think we might want to keep the tests in `multicluster/base` until they are all pulled over. Thoughts on that?

Reply:
I did try pulling the other three tests from `multicluster/base` and  `TestCrossClusterLoadbalancing` and `TestClusterLocalServic` passed while ` TestTelemetry` didn't:
```
2021-08-03T20:18:42.651998Z	info	tf	=== DONE (passed):  Test: 'istiodremote_multicluster[TestTelemetry/telemetry] (27.7283ms)' ===
=== CONT  TestTelemetry/telemetry/config-1->http://config-1:http/
    telemetry.go:54: istio_requests_total not found in stats: map[envoy_cluster_assignment_stale:name:"envoy_cluster_assignment_stale" type:COUNTER metric:<label:<name:"cluster_name" value:"xds-grpc" > counter:<value:0 > >  envoy_cluster_assignment_timeout_received:name:"envoy_cluster_assignment_timeout_received" type:COUNTER metric:<label:<name:"cluster_name" value:"xds-grpc" > counter:<value:0 > >  envoy_cluster_bind_errors:na
```